apps/prairielearn/src/lib/score-upload.ts

```diff
@@ -62,34 +62,38 @@ export async function uploadInstanceQuestionScores(
     let skippedCount = 0;
 
     job.info(`Parsing uploaded CSV file "${csvFile.originalname}" (${csvFile.size} bytes)`);
-    const csvStream = streamifier.createReadStream(csvFile.buffer, {
-      encoding: 'utf8',
-    });
-    const csvConverter = csvtojson({
-      colParser: {
-        instance: 'number',
-        score_perc: 'number',
-        points: 'number',
-        manual_score_perc: 'number',
-        manual_points: 'number',
-        auto_score_perc: 'number',
-        auto_points: 'number',
-        submission_id: 'number',
```

Comment:
There's no need to convert the submission ID to number, so this wasn't included in the conversion.